### ISSUE TITLE
Question: Code documentation on IDE

### ISSUE THREAD

**Issue by redddcyclone** [quarkiverse/quarkus-primefaces#286]
Hello

This is just a question I have. Before I moved to quarkus-primefaces, I used to add the normal PrimeFaces dependency to pom.xml, and with it, code documentation used to appear on my IDE (I used Eclipse before, now IntelliJ, but same thing happens on both), both on .xhtml and .java files, like when hovering a method. With quarkus-primefaces, I don't see it anymore. Is it something that can be fixed on my side or is it unfixable for any reason?

Thanks!

**Comment by tandraschko** [comment]
did you try to import primefaces diretly as dependency?

for me it works fine in IDEA

**Comment by redddcyclone** [comment]
@tandraschko I see now that at least in IDEA my problem was caused by [this issue](https://youtrack.jetbrains.com/issue/IDEA-321739). I'm behind a corporate proxy. Solved it by pulling sources and javadoc manually using ```mvn dependency:sources``` and ```mvn dependency:resolve -Dclassifier=javadoc```.

But though I stopped using Eclipse, I do remember that it used to work with the PrimeFaces dependency but not with quarkus-primefaces... well, that was months ago, though.

Thanks a lot!
